components/Login/UserLogin.js

```diff
@@ -31,7 +31,7 @@ export const UserLogin = (props) => {
   };
 
   return (
-    <View style={Styles.form}>
+    <KeyboardAvoidingView behavior="padding" style={Styles.form}>
```

Comment:
Wondering whether we should wrap our entire app in this as we are going to have more input fields?

Reply:
Not recommended, can cause weird behavior because this component will automatically adjust its height, position, or bottom padding based on the keyboard height